src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stylesObj = {};
+  const lines = sourceString.split(';');
+
+  lines.forEach(function (line) {
+    const part = line.split(':');
+
+    if (part.length === 2) {
+      stylesObj[part[0].trim()] = part[1].trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Instead of using a loop with forEach, consider using more declarative iteration methods like map and filter to transform the array of styles into the desired object format. This approach aligns with functional programming practices and can make the code more concise and readable.